tests/backup/backup_test.go

```diff
@@ -32,7 +32,3 @@ var _ = AfterSuite(func() {
 	//PerformSystemCheck()
 	//ValidateCleanup()
 })
-
-func init() {
-	ParseFlags()
-}
```

Comment:
we must include 
```
func TestMain(m *testing.M) {
	// call flag.Parse() here if TestMain uses flags
	ParseFlags()
	os.Exit(m.Run())
}
```

otherwise, the flags won't be parsed.

Reply:
Yes, it should be added. It will be added as part of tests that @vinayakshnd is adding.